domains/auth/Auth.API/API/Repository/TokenStorage.cs

```diff
@@ -11,5 +13,40 @@ public string GetIdTokenByOpaqueToken(string token)
         {
             throw new NotImplementedException();
         }
+
+        public bool InternalTokenValidation(InternalToken interalToken)
+        {
+            if (interalToken == null)
+            {
+                return false;
+            }
+
+            if (interalToken.Issued > DateTime.UtcNow || interalToken.Expires < DateTime.UtcNow)
```

Comment:
@C-Christiansen Maybe we can extract this boolean to a private method tobe more descriptive on what it does?

Reply:
Extracted to fluentvalidation